examples/prom/nonlinear_elasticity_global_rom.cpp

```diff
@@ -328,7 +328,7 @@ void MergeBasis(const int dimFOM, const int nparam, const int max_num_snapshots,
     generator.endSamples(); // save the merged basis file
 
     int cutoff = 0;
-    generator.finalSummary(1e-7, cutoff, "mergedSV_" + name + ".txt");
+    generator.finalSummary(1e-7, cutoff, "mergedSV_" + name + ".txt",0, squareSV);
```

Comment:
Sorry for asking this again.. there is one more comma in this line. 

Otherwise everything looks good to me!

Reply:
@siuwuncheung Cool, in the latest commit, it's fixed.

Reply:
Thanks @axla-io. Now this gets enough approvals. Please merge this PR to the master when you get a chance.